### PR TITLE
feat(mev-relay): validate bid request

### PR DESCRIPTION
**Pull request by chirag-bgh**
- Implements the function `validate_bid_request`

### CODE REVIEW

**Comment by PatStiles** [comment]
@chirag-bgh  I am working on this finishing the relay for my EPF. Would you mind looking for something else to finish?

**Comment by chirag-bgh** [comment]
> @chirag-bgh I am working on this finishing the relay for my EPF. You can see I previously opened a WIP pr. Would you mind looking for something else to finish?

Hey, you can implement the other parts left!

File: mev-relay-rs/src/relay.rs [hunk 1]
```diff
@@ -26,14 +26,31 @@ use std::{collections::HashMap, ops::Deref, sync::Arc};
...
-    // TODO validations
-
-    // verify slot is timely
+fn validate_bid_request(
+    bid_request: &BidRequest,
+    context: &Context,
+    validator_registry: &ValidatorRegistry,
+) -> Result<(), Error> {
+    let timestamp = get_current_unix_time_in_secs();
+    let current_slot =
```
Comment (chirag-bgh) [inline mev-relay-rs/src/relay.rs:35]: @ralexstokes is this correct for `current_slot`?

**Comment by ralexstokes** [comment]
https://github.com/ralexstokes/mev-rs/pull/113#issuecomment-1711794280

**Pull request closed without merging**